intera_examples/scripts/camera_display.py

```diff
@@ -71,7 +71,12 @@ def main():
 
     print("Initializing node... ")
     rospy.init_node('camera_display', anonymous=True)
-    camera = intera_interface.Cameras()
+    try:
+        camera = intera_interface.Cameras()
+    except OSError as e:
```

Comment:
Is there anything additionally useful in the error message which could be printed?

Reply:
Not without modifying the Cameras class to throw custom error. Definitely an improvement that can be considered for the future.